### PR TITLE
ADAPT-378 Style global brand bar and masthead with logo

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Style global brand bar and masthead with logo

# Review By (Date)
- will update

# Urgency
- medium

# Steps to Test

1. Pull this branch
2. Clear render cache
3. View local site's masthead/brand bar

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [ADAPT-378](https://stanfordits.atlassian.net/browse/ADAPT-378)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by rebeccahongsf** [comment]
@katrialesser this PR is ready for review. 😄 

~I do have a bit of confusion as to what the hover state for the navigation should be. Should the hover state be just the color `#8C1515` or should it be the color AND include the underline?~
(see [Figma](https://www.figma.com/file/TXNBECAGcEkZ3vwc6xcZ3g/SAA-Design-System?node-id=876%3A3736))

~Currently, I styled the hover state to only change the colors of the inactive tab text to `#8C1515` and the active tab text and underline to `#8C1515`.~

~Also we might need to swap out the search bar icon for a new svg file in order to change the color to `#2E2D29`.~ 🤔 

Confusion has been resolved!

**Comment by rebeccahongsf** [comment]
I've tweaked the search icon button size a bit from `12px` to `18px` since it seems incredible small. Let me know your thoughts and if I should revert back to `12px` 😄

File: src/scss/components/main-nav/_main-nav.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.su-multi-menu {
+  .su-multi-menu__nav-toggle--right {
+    &:hover::before,
+    &::before {
+      color: transparent;
+      background-color: transparent;
+    }
+
+    &[aria-expanded="true"] {
+      color: #e50808;
```
Comment (katrialesser) [inline src/scss/components/main-nav/_main-nav.scss]: Colors should all be in variable names in the scss files. It should auto complete in php storm for you.

If there is ever a color not in there or our saa colors file we can add it to the saa colors file.

File: src/scss/components/main-nav/_main-nav.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        }
+      }
+
+      &.su-multi-menu__item--active-trail {
+        .su-multi-menu__link {
+          color: #e50808;
+
+          &::before {
+            color: #e50808;
+            background-color: #e50808;
```
Comment (katrialesser) [inline src/scss/components/main-nav/_main-nav.scss]: The :before element is being removed from the active li on hover; I think we need to adjust the javascript that is controlling this so it doesn't remove the :before element.

**Review by katrialesser — CHANGES_REQUESTED** [review]

**Review by katrialesser — APPROVED** [review]

**Pull request merged**